### PR TITLE
Add Cisco-Meraki integration's Readme

### PR DESCRIPTION
**Pull request by ShacharKidor**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/content-docs/issues/790

## Description
Added a readme file to CISCO Meraki Integration in order to add the integration to the Reference Docs index. 

## Minimum version of Cortex XSOAR
- [x] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/cisco-meraki/Integrations/cisco-meraki/cisco-meraki.yml [hunk 1]
```diff
@@ -622,6 +623,13 @@ script:
       description: Organization id
     - name: headers
       description: Table's headers to be shown by order
+    outputs:
+    - contextPath: status
+      description: license status
+      type: String
+    - contextPath: expirationDate
+      description: license expiration date
+      type: String
```
Comment (ShahafBenYakir) [inline Packs/cisco-meraki/Integrations/cisco-meraki/cisco-meraki.yml:632]: This was missing? Did you catch this using the SDK command?

File: Packs/cisco-meraki/ReleaseNotes/1_0_1.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Cisco Meraki
+- Added a readme file of Cisco-Meraki integration.
```
Comment (ShahafBenYakir) [inline Packs/cisco-meraki/ReleaseNotes/1_0_1.md:4]:
```suggestion
- Added documentation for Cisco-Meraki integration.
```

**Review by ShahafBenYakir — COMMENTED** [review]
Let's wair for @yaron-libman teams comments

File: Packs/cisco-meraki/Integrations/cisco-meraki/cisco-meraki.yml (hunk 1 above)
Comment (ShacharKidor) [reply Packs/cisco-meraki/Integrations/cisco-meraki/cisco-meraki.yml:632]: It was missing. I caught this after executing this command manually and saw that in the readme file the output file of the command is empty.

**Comment by ShirleyDenkberg** [comment]
@ShahafBenYakir Doc review completed.  
@ShacharKidor There were many commands in the README for which there were no examples.

**Comment by ShacharKidor** [comment]
> @ShahafBenYakir Doc review completed.
> @ShacharKidor There were many commands in the README for which there were no examples.

Yes I know, I couldn't find any more examples that returns results. 
Found examples only for the following 3 commands: 
- meraki-fetch-organizations
- meraki-get-organization-license-state
- meraki-fetch-networks

**Review by ShahafBenYakir — APPROVED** [review]
Try to add the missing examples. Besides this it looks good to me

**Comment by ShacharKidor** [comment]
> Try to add the missing examples. Besides this it looks good to me

in order to get command examples for the documentation of this integration, we need to run the commands and get some outputs but the instance for CISCO Meraki is missing some items/elements:
Inventory (in Organization)
Devices (in Network)
Clients (in Network)

@meital-aizen is helping me with that. After the required items will be added to our instance I'll add the missing command examples.

**Pull request merged**